package.json

```diff
@@ -91,9 +91,9 @@
   },
   "dependencies": {
     "@tanstack/react-query": "^4.28.0",
-    "seamapi": "^5.19.0-alpha.2",
     "classnames": "^2.3.2",
-    "dayjs": "^1.11.7",
+    "luxon": "^3.3.0",
```

Comment:
Should we remove `dayjs`, and `ms` if we're using `luxon`?

Reply:
Ahh, sorry I see it IS removed.

Reply:
We could use dayjs or Luxon. Main reason for adding it here was to see how it works as an alternative and start a discussion. If we are ok with Luxon though then no need to hash it out, but it would be my first choice.